content/en/tutorials/touch-joypad.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Touchscreen Joypad Controls
+layout: tutorial-page.hbs
+tags: input,ui
+thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/1007506/2F5E56-image-75.jpg
+---
+
+<iframe loading="lazy" src="https://playcanv.as/p/kvE0iJWc/" title="Touchscreen Joypad Controls"></iframe>
+
+[Click here to see the project][project-link].
+
+## Overview
+
+This tutorial shares a Touchscreen Joypad library that can be installed and customized for any project that needs touch screen controls.
+
+We recommend reading about the [PlayCanvas UI system][playcanvas-ui] if you are not familiar with it as it to render and position the controls.
+
+![][preview]
+
+## How to install
+
+Open the [example project][project-link], right click on the folder 'touch-joypad' and click on 'Copy'.
+![][copy-folder]
+
+Open your project, right click in the assets panel and click on 'Paste'
+![][paste-folder]
+
+## Adding your joystick
+
+In the folder we've just pasted, open the 'templates' folder and there will be 4 preconfigured templates for the joysticks to cover the most common use cases.
+
+* **Left/Right Fixed Touch Joystick** - A fixed joystick that is anchored to the bottom left/right of the screen corners. Recommended for quick reaction games where an instant response is expected such as a retro platforming game.
+* **Left/Right Half Touch Joystick** - A joystick where the input area the left/right half of the screen and the joystick will move to where you touch in that area and them requires dragging for movement. Recommended for when 'pushing' or 'pulling' an object in the application.
```

Comment:
and then?

Reply:
What do you mean 'and then' 😅 'Left/Right Fixed Touch Joystick' are two separate templates if that's what you mean?